client/components/select-dropdown/index.jsx

```diff
@@ -207,12 +207,20 @@ class SelectDropdown extends Component {
 			} );
 		}
 
-		let dropdownClassName = classNames( dropdownClasses );
-		let selectedText = this.props.selectedText
-			? this.props.selectedText
-			: result( find(
-				this.props.options, { value: this.state.selected }
+		const dropdownClassName = classNames( dropdownClasses );
+
+		let selectedText = this.props.selectedText;
```

Comment:
Nice. This also pairs well with the idea of having unit tests...

I will update the pull request; by the way what is your workflow to update a pull request? Is it ok if I simply push a new commit on this branch?